plugins/auth-backend/src/providers/cloudflare-access/provider.ts

```diff
@@ -349,6 +390,13 @@ export const cfAccess = createAuthProviderIntegration({
   }) {
     return ({ config, resolverContext }) => {
       const teamName = config.getString('teamName');
+      const serviceTokensConfig = config.getOptionalConfig('serviceTokens');
```

Comment:
Any additions to the config format needs to also be added to the package's `config.d.ts` file. Especially so, since these absolutely must be marked as `@visibility secret` so that they don't accidentally leak and become a security risk.

With that in mind, would it make sense to instead make the format be a list?

```yaml
serviceTokens:
  - token: ey.ey.ey
    subject: email@company.com
```

to make it more explicit and easier to put a harsher visibility on just one of the items in there. And did I understand that this was actually a re-mapping? Should it be `forceEmail: email@company.com` or something that describes the purpose of that setting?

Reply:
Thanks for calling the config.d.ts out, not sure how I completely forgot to do that. Though I'm not certain it's actually a security risk, these are just token IDs, not secrets. But better safe than sorry, they shouldn't be needed in the frontend anyway.

I do like the list of objects structure a lot better, so I've changed that. I'm still on the fence about the `subject` name. It's only a re-mapping in the sense that the tokens are missing any mapping at all so we have to add one at this layer (very annoying, I wish Cloudflare would just support Service Token identities and I could this PR). But I'm happy to change it to `forceEmail` or any other name if you like.